Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift

```diff
@@ -825,7 +823,28 @@ private class MacroApplication<Context: MacroExpansionContext>: SyntaxRewriter {
       }
     }
 
-    return .init(newItems)
+    /// Returns an leading trivia for the member blocks closing brace.
+    /// It will add a leading newline, if there is none.
+    var leadingTriviaForClosingBrace: Trivia {
+      if (node.members.isEmpty || newItems.isEmpty) && newItems.isEmpty {
```

Comment:
Reading the code for the 4-th time I just realized. Isn’t this logically equivalent to

```suggestion
      if newItems.isEmpty {
```